src/components/PostsList.tsx

```diff
@@ -2,85 +1,65 @@
+import { Post } from '../types/Post';
+import classNames from 'classnames';
 
-export const PostsList: React.FC = () => (
-  <div data-cy="PostsList">
-    <p className="title">Posts:</p>
+interface Props {
+  posts: Post[];
+  selectedPost: Post | null;
+  setSelectedpost: React.Dispatch<React.SetStateAction<Post | null>>;
+}
 
-    <table className="table is-fullwidth is-striped is-hoverable is-narrow">
-      <thead>
-        <tr className="has-background-link-light">
-          <th>#</th>
-          <th>Title</th>
-          {/* eslint-disable-next-line jsx-a11y/control-has-associated-label */}
-          <th> </th>
-        </tr>
-      </thead>
+export const PostsList: React.FC<Props> = ({
+  posts,
+  selectedPost,
+  setSelectedpost,
```

Comment:
The prop `setSelectedpost` should be renamed to `setSelectedPost` to maintain consistency with the naming convention used for state variables and props.